### PR TITLE
Update docs README.md

### LINKED ISSUE #77: Update doc homepage
The current homepage is still with old tagline and key points:

```
Rapid feature integration
Plug features into any application - we are even using MESG to build MESG itself.

Multi-language applications
MESG mixes a gRPC API and a container based module approach to offer the most flexibility to developers.

Event-driven architecture
Event-driven architecture allows any services to produce events and react to other with loosely coupled dependencies.

Autonomous workflows
MESG features flow-based programming to create reactive applications with endless possibility.

Hybrid decentralized applications
MESG’s decentralized network will enable you to run centralized services in decentralized trustless environments.

Scalable applications
The mix of an Event-driven architecture and a decentralized network offers an high scalability and redundancy infrastructure
```

@willcreatedit can you take a look and update it?

### PR DESCRIPTION
**Pull request by williamzmorgan**
Ok - here is an attempt. I think either "decentralized executions", or "multi-language applications" should be replaced with "Keep it Simple" - to explain Orchestrator. Thoughts?

closes https://github.com/mesg-foundation/docs/issues/77

### CODE REVIEW

**Comment by williamzmorgan** [comment]
I also want this to be very geeky. So please enhance it with high-level language, if you can see opportunities to do so.

File: README.md [hunk 1]
```diff
@@ -4,16 +4,16 @@ heroImage: ./logo.png
...
-  - title: Hybrid decentralized applications
-    details: MESG’s decentralized network will enable you to run centralized services in decentralized trustless environments.
+  - title: Don’t repeat yourself
+    details: Any features, whether centralized or decentralized, can be reused within any application. No heavy reconfigurations required.
+  - title: Stay in control
+    details: Choose exactly how and where your services run, whether on trustless networks, trusted cloud providers, or private servers in any combination. 
   - title: Scalable applications
     details: The mix of an Event-driven architecture and a decentralized network offers an high scalability and redundancy infrastructure
+  - title: Decentralized executions
+    details: The upcoming decentralized network will enable applications and services to run and execute from entirely within a decentralized cloud.
```
Comment (NicolasMahe) [inline README.md:18]: I like this one but i would suggest to use process instead of execution.
There is nothing about the process anymore.

**Comment by NicolasMahe** [comment]
@willcreatedit the header title is in this file: https://github.com/mesg-foundation/docs/blob/will-docs-home/.vuepress/config.yml#L2

**Comment by williamzmorgan** [comment]
Cool - updated the header tagline and changed "decentralized executions" to "decentralized processes"

**Review by NicolasMahe — APPROVED** [review]

**Review by antho1404 — APPROVED** [review]

**Pull request merged**